CONTRIBUTING.md

```diff
@@ -0,0 +1,111 @@
+# Contributing to `runme`
+
+**Thank you for your interest in `runme`. Your contributions are highly welcome.**
+
+There are multiple ways of getting involved:
+
+- [Report a bug](#report-a-bug)
+- [Suggest a feature](#suggest-a-feature)
+- [Contribute code](#contribute-code)
+
+## Report a bug
+
+Reporting bugs is one of the best ways to contribute. Before creating a bug report, please check that an [issue](/issues) reporting the same problem does not already exist. If there is such an issue, you may add your information as a comment.
```

Comment:
```suggestion
Reporting bugs is one of the best ways to contribute. Before creating a bug report, please check that an [issue](../../issues) reporting the same problem does not already exist. If there is such an issue, you may add your information as a comment.
```

Reply:
@christian-bromann 

Would the link not be top level since the file is at top level for the repo? The href is copied directly from the CONTRIBUTING file in `vscode-runme`